windows-rdp/powershell-installation-script.tftpl

```diff
@@ -0,0 +1,85 @@
+function Set-AdminPassword {
+    param (
+        [string]$adminPassword
+    )
+    # Set admin password
+    Get-LocalUser -Name "${admin_username}" | Set-LocalUser -Password (ConvertTo-SecureString -AsPlainText $adminPassword -Force)
+    # Enable admin user
+    Get-LocalUser -Name "${admin_username}" | Enable-LocalUser
+}
+
+function Configure-RDP {
+    # Enable RDP
+    New-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server' -Name "fDenyTSConnections" -Value 0 -PropertyType DWORD -Force
+    # Disable NLA
+    New-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -Name "UserAuthentication" -Value 0 -PropertyType DWORD -Force
+    New-ItemProperty -Path 'HKLM:\System\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp' -Name "SecurityLayer" -Value 1 -PropertyType DWORD -Force
+    # Enable RDP through Windows Firewall
+    Enable-NetFirewallRule -DisplayGroup "Remote Desktop"
+}
+
+function Install-DevolutionsGateway {
+# Define the module name and version
+$moduleName = "DevolutionsGateway"
+$moduleVersion = "2024.1.5"
```

Comment:
Yeah, just because the DOM selectors in the JavaScript code are so specific, it felt really important to pin Devolutions Gateway to a specific version, to avoid breaking changes as the Devolutions team refactors their own UI

It would be nice to expose it as a mutable value, but I feel like it might be asking for trouble, and could cause things to break unexpectedly